app/index.html

```diff
@@ -48,17 +48,20 @@ <h1 class="docs-title" id="page-title">{{ page.title }}</h1>
 <main id="main" class="docs-text">
 
     <h2 id="welcome">Welcome to H2O!</h2>
+    
+    <h3 id="what">What is H2O?</h3>
+    <p>H2O is a website that helps faculty authors make casebooks that are free, simple to modify, and easy to access and print. H2O is offered by the Harvard Law School Library, through its <a href="https://lil.law.harvard.edu" target="_blank">Library Innovation Lab</a>.</p>
 
     <h3 id="why">Why did we build H2O?</h3>
     <p>The casebook - annotated collections of court cases - has been a cornerstone of legal education for over 150 years. But casebooks today are heavy, expensive and way too difficult to use.</p>
     <p>We believe casebooks can be better for authors and students. Authors should be able to easily create or customize their own casebooks online. And students should be able read the casebooks online or in print for little or no cost.</p>
 
-    <h3 id="what">What is H2O?</h3>
-    <p>H2O is a website that helps faculty authors make casebooks that are free, simple to modify, and easy to access and print. H2O is offered by the Harvard Law School Library, through its <a href="https://lil.law.harvard.edu" target="_blank">Library Innovation Lab</a>.</p>
-
     <h3 id="how">How does H2O work?</h3>
-    <p>H2O gives authors access to the entire database of U.S. court decisions made available through the <a href="https://case.law" target="_blank">CaseLaw Access Project</a>, another initiative of the Library Innovation Lab. Authors can pull any of those cases into their casebooks, together with their own texts and links to external sources on the web. These casebooks, in turn, are free to access and can be copied, shared and adapted under a <a href="https://creativecommons.org/licenses/by-nc-sa/3.0/" target="_blank">CC BY-NC-SA 3.0 license</a>. In addition, H2O's software is open source and available on <a href="https://github.com/harvard-lil/h2o" target="_blank">Github</a>. </p>
+    <p>You can use H2O to create casebooks, reading lists, course outlines, syllabi, and more. H2O gives authors access to the entire database of U.S. court decisions made available through the <a href="https://case.law" target="_blank">CaseLaw Access Project</a>, another initiative of the Library Innovation Lab. Authors can pull any of those cases into their casebooks, together with their own texts and links to external sources on the web. These casebooks, in turn, are free to access and can be copied, shared and adapted under a <a href="https://creativecommons.org/licenses/by-nc-sa/3.0/" target="_blank">CC BY-NC-SA 3.0 license</a>. In addition, H2O's software is open source and available on <a href="https://github.com/harvard-lil/h2o" target="_blank">Github</a>. </p>
 
+    <h3>How can I use H2O?</h3>
```

Comment:
"What can I make with H2O?" seems like a slightly more precise phrasing for the given answer.

Reply:
Just created new PR!